### ISSUE TITLE
Warning with default settings

### ISSUE THREAD

**Issue by maicol07** [nuuday/github-changelog-action#1]
I get this warning with default settings even if my config folder exists:

```
Warning: git-chlog configuration was not found, skipping changelog generation.
```

Repo: https://github.com/maicol07/flarum-ext-sso

**Comment by ahaasler** [comment]
Hi, I was having the same error on Sunday and found this issue, I resolved it using master instead of v1.0.0:

https://github.com/ahaasler/dotfiles/blob/3b3c0122a50c80fd18f53a15f05e576461b82d5b/.github/workflows/build.yml#L86-L92

I think that using `nuuday/github-changelog-action@master` works directly in your case, could you give it a try?

**Comment by maicol07** [comment]
> 
> 
> Hi, I was having the same error on Sunday and found this issue, I resolved it using master instead of v1.0.0:
> 
> [ahaasler/dotfiles@`3b3c012`/.github/workflows/build.yml#L86-L92](https://github.com/ahaasler/dotfiles/blob/3b3c0122a50c80fd18f53a15f05e576461b82d5b/.github/workflows/build.yml#L86-L92)
> 
> I think that using `nuuday/github-changelog-action@master` works directly in your case, could you give it a try?

@ahaasler doesn't work

**Comment by maicol07** [comment]
Finally fixed this! I was missing a lot of things on how Github Actions works. Here is my workflow: https://github.com/maicol07/flarum-ext-sso/blob/master/.github/workflows/changelog.yml
